pkg/controller.v1/tensorflow/controller.go

```diff
@@ -248,7 +248,12 @@ func (tc *TFController) processNextWorkItem() bool {
 	if err != nil {
 		if err == errNotExists {
 			logger.Infof("TFJob has been deleted: %v", key)
-			tfJobsDeletedCount.WithLabelValues(tfJob.Namespace).Inc()
+			namespace, _, keyerr := cache.SplitMetaNamespaceKey(key)
```

Comment:
These code are simalar to `getTFJobFromName` in `syncTFJob` function, can we remove these code?
![image](https://user-images.githubusercontent.com/9292128/124083916-008fea00-da81-11eb-9f8d-eef8d2bf144d.png)

![image](https://user-images.githubusercontent.com/9292128/124084421-8ad84e00-da81-11eb-9c5d-650ed0a7ecb1.png)

Reply:
Can you please explain more about it? Thanks

Reply:
@D0m021ng Both checks are different. We need to verify if there is case for  `tc.getTFJobFromKey(key)` returns no error while `cache.SplitMetaNamespaceKey` returns error. We can do it in a separate PR

Reply:
Thanks for the explanation.